documentation/src/components/CodeBlock.js

```diff
@@ -0,0 +1,49 @@
+/* eslint react/jsx-key: 0 */
+import React from 'react'
+import Highlight, {defaultProps} from 'prism-react-renderer'
+import {LiveProvider, LiveEditor, LiveError, LivePreview} from 'react-live'
+import {mdx} from '@mdx-js/react'
+
+export default ({children, className, live, render}) => {
```

Comment:
What were your changes in that swizzled file? @larsmoa

Reply:
Oops, unnecessary file. Removed.